### PR TITLE
Remove dynamic fixing switches in GK on phala

### PR DESCRIPTION
**Pull request by kvinwang**
As we decided to not ever launch pruntime v2 on khala, these switches are useless in the future.
Removing them can make it simpler to launch pruntime on phala.
Two changes:
- Removing the bug-fixing switches in GK.
- Syncing the TokenomicParameters from chain storage upon GK launching.

### CODE REVIEW

File: crates/phactory/src/system/gk.rs [hunk 1]
```diff
@@ -2248,11 +2150,11 @@ pub mod tests {
         let report = r.gk.egress.drain_working_info_update_event();
         assert_eq!(
             FixedPoint::from_bits(report[0].settle[0].payout),
-            fp!(14.69197867920878555043)
+            fp!(576424.184820453732333905)
```
Comment (kvinwang) [inline crates/phactory/src/system/gk.rs:2153]: The entire budget is paid to the single worker here.
We need to carefully carry the reward budget from khala to phala gradually.

Comment (h4x3rotab) [reply crates/phactory/src/system/gk.rs:2160]: That's by design. Once Phala and Khala are live at the same time, we plan to run a Phat Contract to allocate budget between the two. The allocation will be calculated based on the workers (and their scores, etc). I think that's what you mean "carry the budget gradually". Am I right?

File: crates/phactory/src/system/mod.rs [hunk 2]
```diff
@@ -860,13 +860,19 @@ impl<Platform: pal::Platform> System<Platform> {
...
         );
-        let gatekeeper = gk::Gatekeeper::new(
+        let mut gatekeeper = gk::Gatekeeper::new(
             master_key_history,
             block.recv_mq,
             block
                 .send_mq
                 .channel(MessageOrigin::Gatekeeper, master_key.into()),
         );
+        if let Some(params) = block.storage.tokenomic_parameters() {
```
Comment (h4x3rotab) [inline crates/phactory/src/system/mod.rs:870]: Does it update the tokenomic parameter at every block?

Comment (kvinwang) [reply crates/phactory/src/system/mod.rs:870]: No, it's a one-shot operation on the GK launch. Later updates will keep using mq.

File: crates/phactory/src/system/gk.rs (hunk 1 above)
Comment (kvinwang) [reply crates/phactory/src/system/gk.rs:2160]: > Am I right?

Yes. Absolutely.

**Pull request merged**